src/test/java/com/americanexpress/span/core/database/comptest/SPExecutorCompTest.java

```diff
@@ -201,7 +209,7 @@ public void testGetMethod() throws SQLException {
         assertEquals(inputObject.getInBooleanObj(), outputObject.getOutBooleanObj());
 
         assertEquals(2, outputObject.getListRSObj().size());
-        assertEquals(inputObject.getInTimestamp(), outputObject.getOutTimestamp());
+        assertEquals(inputObject.getInTimestamp().truncatedTo(ChronoUnit.SECONDS), outputObject.getOutTimestamp().truncatedTo(ChronoUnit.SECONDS));
```

Comment:
I was getting assertion Error ( please see below) due to mismatch in milliseconds. Thus I used truncatedTo seconds.

java.lang.AssertionError: 
Expected :2020-12-02T12:40:23.670348800
Actual   :2020-12-02T12:40:23.670348
<Click to see difference>